querydsl-sql/src/test/java/com/querydsl/sql/AbstractSQLTemplatesTest.java

```diff
@@ -70,19 +70,19 @@ public void union() {
 
         if (templates.getDummyTable() == null) {
             assertEquals(
-                    "(select 1 as col1)\n" +
```

Comment:
this should be changed in the subclass, look at the test failures
alternatively parametrize it better